src/node_process_methods.cc

```diff
@@ -53,6 +53,7 @@ using v8::Value;
 
 namespace per_process {
 Mutex umask_mutex;
+bool emitted_insecure_umask_warning = false;
```

Comment:
@addaleax ... just thinking... should this be hung off `Environment` rather than process-scoped?

Reply:
@jasnell That would definitely be more consistent with other runtime deprecations, but at the same time I personally don’t have a strong opinion

Reply:
I would certainly like to err on the side of consistency. here :-)